blimflix/client/app/providers/seen.factory.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+  'use strict';
+
+  /**
+   * Chain to fetch module
+   *   https://github.com/johnpapa/angular-styleguide#style-y022
+   */
+  angular
+    .module('blimFlix.providers')
+    .factory('seen', seen);
+    seen.$inject = ['$resource'];
+
+  /**
+   * Avoid anonymous functions as callbacks
+   *   https://github.com/johnpapa/angular-styleguide#style-y024
+   */
+  /* @ngInject */
+  function seen($resource) {
+
+    return $resource('http://localhost:3333/users/:id/seen/:idM',{id:'@id', idM:'@idM'}, {
+      'query': { method: 'GET', isArray: true},
+      'save': { method: 'POST' },
+      'remove': { method:'DELETE'}
```

Comment:
Just a suggestion: When you have variable/object assign value is a good practice code like this example, look the right values are align with the same spaces after the ":", this way is more readily :) 

'query':    { method: 'GET', isArray: true},
'save':      { method: 'POST' },
'remove': { method:'DELETE'}